Remotion/Core/Collections.Caching.UnitTests/ConcurrentCacheTest.cs

```diff
@@ -26,6 +27,7 @@
 namespace Remotion.Collections.Caching.UnitTests
 {
   [TestFixture]
+  [SuppressMessage("Usage", "RMCORE0001: Use Safecontext instead of typical API")]
```

Comment:
I think we should use `pragma` instead instead of `SupressMessage` in this test class. The idea is that any newly added code is reminded of the safe context problem instead of having the message already supressed. These tests probably won't have any SafeContext problems to begin with but I think it is cleaner this way.

The SafeContext related tests on the other hand should stay with `SupressMessage`. This is were almost every test would create a problem making pragma unattractive. Also since the tests are SafeContext related they any changes would need to consider SafeContext anyway so there is no need for the extra reminder